tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -9740,99 +9749,141 @@ TEST_F(ExecutionTest, DynamicResourcesUniformIndexingTest) {
       std::make_shared<st::ShaderOpSet>();
   st::ParseShaderOpSetFromStream(pStream, ShaderOpSet.get());
   st::ShaderOp *pShaderOp =
-      ShaderOpSet->GetShaderOp("DynamicResourcesUniformIndexing");
+      ShaderOpSet->GetShaderOp("DynamicResourcesDynamicIndexing");
 
   bool Skipped = true;
 
   //D3D_SHADER_MODEL TestShaderModels[] = {D3D_SHADER_MODEL_6_0}; // FALLBACK
-  D3D_SHADER_MODEL TestShaderModels[] = {D3D_SHADER_MODEL_6_6};
+  D3D_SHADER_MODEL TestShaderModels[] = {D3D_SHADER_MODEL_6_0, D3D_SHADER_MODEL_6_6};
```

Comment:
I am a bit torn with what to do here.  I don't think we should be checking in the test in a state that will silently pass without SM 6.6 support, without some special selection for the fallback mode.  That selection could be made with a define, or maybe even with a test parameter.  We should standardize this mechanism across tests somehow as well.

Perhaps the cleanest approach is to add a method like `bool ExecutionTest::IsFallbackPathEnabled()` that does something like:
```
// Enable fallback paths with: /p:"EnableFallback=1"
UINT EnableFalbackValue = 0;
WEX::TestExecution::RuntimeParameters::TryGetValue(L"EnableFallback", EnableFallbackValue);
return EnableFalbackValue != 0;
```
In a future change, we can update other tests that silently support fallback paths or use a define to use this instead.